packages/react-magma-dom/src/components/Dropdown/DropdownButton.tsx

```diff
@@ -131,19 +131,21 @@ export const DropdownButton = React.forwardRef<
     : ButtonIconPosition.right;
 
   return (
-    <StyledIconButton
-      {...other}
-      aria-expanded={context.isOpen}
-      aria-haspopup="true"
-      icon={icon}
-      iconPosition={iconPositionToUse}
-      id={context.dropdownButtonId.current}
-      isInverse={context.isInverse}
-      onClick={handleClick}
-      ref={ref}
-      theme={theme}
-    >
-      {children}
-    </StyledIconButton>
+    <div data-testid={'dropdownButton-wrapper'} ref={context.setReference}>
```

Comment:
Is the test id needed? I like to avoid hardcoded test ids whenever possible because our adopters will then end up with repeated IDs in their codebase if they have more than one Dropdown on a page. 

If we are including these here for our own unit tests, we should consider passing it directly to the component (`<DropdownSplitButton aria-label="Split" data-testId="myUniqueId">`) or if it doesn't get passed to the right  element on the DOM, adding testId props.

Reply:
Deleted!